### PR TITLE
Star 165 takeover focus lock api

### PR DESCRIPTION
**Pull request by tristanjasper**
### Purpose 🚀
Ensuring focusLock api in takeover component is consistent with SidePanel and Modal component

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [ ] MINOR (backward compatible) change to _these packages_
- [x] PATCH (bug fix) change to SidePanel, Modal, Takeover

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/star-165-takeover-focusLock-api

### Screenshots 📸
_optional but highly recommended_

### References 🔗
https://aclgrc.atlassian.net/browse/STAR-165

### CODE REVIEW

File: packages/Modal/src/Modal.js [hunk 1]
```diff
@@ -71,8 +71,8 @@ const Modal = props => {
       onClose={onClose}
       onAfterOpen={onAfterOpen}
       onAfterClose={onAfterClose}
-      {...overlayProps}
       focusLockOptions={focusLockOptions}
+      {...overlayProps}
```
Comment (tristanjasper) [inline packages/Modal/src/Modal.js]: I would think these props should be applied to override the focusLockOptions....

File: packages/Takeover/src/Takeover.styles.js [hunk 2]
```diff
@@ -3,6 +3,19 @@ import tokens from "@paprika/tokens";
...
+  flex-direction: column;
+  height: 100%;
+
+  &::before,
+  &::after {
+    content: "";
+    display: block;
+  }
+`;
+
```
Comment (tristanjasper) [inline packages/Takeover/src/Takeover.styles.js]: I _think_ these styles are correct, although the `modal` components styles also have a `flex-basis` style which I'm unsure is needed here - https://github.com/acl-services/paprika/blob/master/packages/Modal/src/Modal.styles.js#L48

File: packages/Takeover/src/components/FocusLock/FocusLock.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  returnFocus: true,
+};
+
+const FocusLock = props => <FocusLockLibrary {...props} />;
+
+FocusLock.propTypes = propTypes;
+FocusLock.defaultProps = defaultProps;
+FocusLock.displayName = "Takeover.FocusLock";
+
+export default FocusLock;
```
Comment (tristanjasper) [inline packages/Takeover/src/components/FocusLock/FocusLock.js]: I'm wondering if perhaps this component should be somehow be a shared import or even its own package, but not completely convinced on where it belongs as it is the same component across sidepanel, takeover and modal now

Comment (nahumzs) [reply packages/Takeover/src/components/FocusLock/FocusLock.js]: Totally fine, duplication is not intrinsically bad, for this scenario I think we are fine.

File: packages/Modal/src/Modal.js (hunk 1 above)
Comment (nahumzs) [reply packages/Modal/src/Modal.js]: why would be that? isn't we are having `<Modal.FocusLock  />` in charge of overriding the props?

File: packages/Takeover/README.md [hunk 4]
```diff
@@ -1,31 +1,11 @@
-# Takeover
```
Comment (nahumzs) [inline packages/Takeover/README.md]: why are you removing the basic Readme example? any specific need to do this? the change log should be a different file

Comment (tristanjasper) [reply packages/Takeover/README.md]: Ah wrong file removed.

File: packages/Modal/src/Modal.js (hunk 1 above)
Comment (tristanjasper) [reply packages/Modal/src/Modal.js]: Ok, I see, I wasn't sure of the priority

**Pull request merged**